index.js

```diff
@@ -70,10 +70,11 @@ console.info(`Will upload ${files.length} files to:\n${webRoot}\n`);
 files.forEach(file => {
   let fileOptions = {
     validation: 'crc32c',
-    metadata: {
-      cacheControl: 'no-cache',
-      contentType: mime.lookup(file)
-    },
+    metadata: Object.assign(
+      {},
+      metadata,
+      {contentType: mime.lookup(file)}
```

Comment:
Yes can do that. But I don't think it is very useful because in most cases you have several types and we can only configure one here. But for some cases it's definitely the more flexible way.